Spigot-Server-Patches/0728-Add-corner-getter-to-WorldBorder.patch

```diff
@@ -0,0 +1,39 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: TwoLeggedCat <80929284+TwoLeggedCat@users.noreply.github.com>
+Date: Wed, 12 May 2021 20:07:13 -0500
+Subject: [PATCH] Add corner getter to WorldBorder
+
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java b/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
+index 31f197ded3335cf6615a7e7d4e502323daad13e4..5d7dd3269bf6f2e07b0d91bd439d8ae0102e1c99 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftWorldBorder.java
+@@ -2,9 +2,12 @@ package org.bukkit.craftbukkit;
+ 
+ import com.google.common.base.Preconditions;
+ import net.minecraft.core.BlockPosition;
++import org.apache.commons.lang.Validate;
++import org.bukkit.Corner;
+ import org.bukkit.Location;
+ import org.bukkit.World;
+ import org.bukkit.WorldBorder;
++import org.jetbrains.annotations.NotNull;
```

Comment:
Use fully qualified class names instead of adding imports, since its just a few imports that aren't used too often.